src/sentry/ingest/consumer/processors.py

```diff
@@ -169,6 +171,21 @@ def process_event(
     else:
         processing_store = event_processing_store
 
+    if (
+        consumer_type == ConsumerType.Transactions or data.get("type") == "transaction"
+    ) and options.get("transactions.stale_dlq_enabled"):
+        # check timestamp
+        timestamp = message.get("timestamp") if message.get("timestamp") else start_time
```

Comment:
not sure if message.get('timesstamp") works need to double check

Reply:
i found in the s[chema repo](https://github.com/getsentry/sentry-kafka-schemas/blob/a31ab4d4537199c72e5ab26689bbd1e8d766e5a5/examples/events/1/errors1.json#L13) that "timestamp" field is on the same level as "type". And in this file the field `type` is being accesed like `data.get("type")`

Reply:
thank you!